app/assets/v2/js/metamask-approval.js

```diff
@@ -2,8 +2,27 @@
 var is_metamask_approved = is_metamask_approved || false;
 var is_metamask_unlocked = is_metamask_unlocked || false;
 
-async function metamaskApproval() {
-  if (window.ethereum && window.ethereum._metamask) {
+window.addEventListener('load', checkProvider);
```

Comment:
This function loads Fortmatic if it was previously loaded (checking local storage). Otherwise it does what this code was doing before (loading metamask if it's available and previously connected, I think).